viewing_party/party.py

```diff
@@ -1,23 +1,165 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    movie = {}
+    movie_set = {title, genre, rating}
+
+    if None in movie_set:
+        return None
+    else:
+        movie["title"] = title
+        movie["genre"] = genre
+        movie["rating"] = rating
+    
+    return movie
+
+def add_to_watched(user_data, movie):
+
+    user_data["watched"].append(movie)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+
+    user_data["watchlist"].append(movie)
+
+    return user_data
+
+def watch_movie(user_data, title):
+
+    for movie in user_data["watchlist"]:
+        if title != movie["title"]:
+            continue
+        else:
+            movie_index = user_data["watchlist"].index(movie)
+            removed_movie = user_data["watchlist"].pop(movie_index)
+            add_to_watched(user_data, removed_movie)

```

Comment:
Rather than having an empty `if` branch, invert the check and move the else logic up to the `if`.
```python
        if title == movie["title"]:
            movie_index = user_data["watchlist"].index(movie)
            removed_movie = user_data["watchlist"].pop(movie_index)
            add_to_watched(user_data, removed_movie)
```

Further, using `index` then `pop` is the equivalent of using `remove`, so we might as well use the single function
```python
        if title == movie["title"]:
            user_data["watchlist"].remove(movie)
            add_to_watched(user_data, movie)
            break # also, add this break so that the iteration halts once we modify the list we are iterating over (danger)
```